DB_VERSION.json

```diff
@@ -0,0 +1 @@
+{"bugs.json": "v1", "commits.json": "v1"}
```

Comment:
Actually, instead of having a separate file, let's just add a third parameter to the db.register function to specify the version

Reply:
But then won't we need to change the arguments to db.register in `bugzilla.py` and `repository.py` every time we update DB?

And adding the argument to db.register would then create a new version file or we would add that version to `DATABASES` dictionary?

Reply:
> But then won't we need to change the arguments to db.register in `bugzilla.py` and `repository.py` every time we update DB?

Yes, but this is no different than updating the VERSION file.

> And adding the argument to db.register would then create a new version file or we would add that version to `DATABASES` dictionary?

Everything stays the same, except that the information about the DB version is not going to be in a file in the repo, but is going to be passed with db.register.

Reply:
Yes, but we need to store the version passed in db.register, right? So that we can check with the local file if our DB need updating?

In db.register, can we do something like:
```
def register(path, url, version):
    DATABASES[path] = [url, version]
```

And in db.download, we can compare the vesions?

Reply:
This is really just an implementation detail, not such a big problem, but I prefer you store a dict in DATABASES[path] (with keys `url` and `version`).